src/core/ext/transport/chttp2/transport/internal.h

```diff
@@ -270,7 +270,6 @@ struct grpc_chttp2_transport {
   grpc_slice_buffer outbuf;
```

Comment:
I can do this change. The main reason I switched away from this solution (which I used in the previous attempts) was the tracing story and because I need read access to t->settings.

To get around this I could duplicate that data, pass it into the api separately, or maybe even have the flow_control* structs hold const pointers back to their transports or streams? Which of those seems cleanest?